mevolib/cluster/__init__.py

```diff
@@ -93,4 +94,13 @@ def get_subsets(method: str, seqfile: str, fileformat: str = 'genbank', *args: t
         for key in iter(set_dict):
             for result in output[1:]:
                 set_dict[key].extend(result[key])
-    return set_dict
\ No newline at end of file
+    return set_dict
+
+
+def main():
```

Comment:
This should not be called `main`, since it is applying the `gene splice` division only. Either you will need to add the option to select the method in the `argparse` or rename the function.